packages/web/src/pages/settings-page/components/desktop/ManagerMode/ManagerModeSettingsModal.module.css

```diff
@@ -0,0 +1,5 @@
+.titleIcon {
+  height: var(--harmony-unit-6);
```

Comment:
i think we prefer harmony-spacing when applicable. so here it would be --harmony-spacing-xl (24px) I believe!

Reply:
Good call, this was copypasta'd from another modal in the same directory. Wasn't worrying about it too much because I expect this to be completely replaced!